src/components/Form.js

```diff
@@ -172,8 +173,19 @@ class Form extends React.Component {
     validate(values) {
         FormActions.setErrors(this.props.formID, null);
         FormActions.setErrorFields(this.props.formID, null);
+
+        // Run any validations passed as a prop
         const validationErrors = this.props.validate(values);
 
+        // Validate the input for html tags. It should superseed any other error
+        _.each(values, (inputValue, inputID) => {
+            if (!inputValue || inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1) {
```

Comment:
I can't quite remember, but are `inputValue` always `{String}`? 

I don't think `.search()` works on numbers. If I'm right maybe we can do something like this instead...

```suggestion
            if (!inputValue || (_.isString(inputValue) && inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1)) {
```